gpswearos/src/main/java/com/android/gpswearos/presentation/MainActivity.kt

```diff
@@ -0,0 +1,104 @@
+/* While this template provides a good starting point for using Wear Compose, you can always
+ * take a look at https://github.com/android/wear-os-samples/tree/main/ComposeStarter and
+ * https://github.com/android/wear-os-samples/tree/main/ComposeAdvanced to find the most up to date
+ * changes to the libraries and their usages.
+ */
+
+package com.android.gpswearos.presentation
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Devices
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.wear.compose.material.Card
+import androidx.wear.compose.material.Text
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            StatusCard(SampleData.statusSample)
+        }
+    }
+}
+
+data class SatelliteStatus(val index: String, val data: String)
+
+@Composable
+fun StatusCard(satStatues: List<SatelliteStatus>) {
+    Card({} , modifier = Modifier
+        .fillMaxWidth()
+        .padding(5.dp)) {
+        Box ( modifier = Modifier
+            .fillMaxSize()
+            .verticalScroll(rememberScrollState()), contentAlignment = Alignment.Center){
+            Row {
```

Comment:
If you use ScalingLazyColumn here and adapt the list of Location elements accordingly, it will handle scaling items on and off the screen as you scroll. See https://youtu.be/B7D3G6tC9n0?t=697.

Reply:
> If you use ScalingLazyColumn here and adapt the list of Location elements accordingly, it will handle scaling items on and off the screen as you scroll. See https://youtu.be/B7D3G6tC9n0?t=697.

Thanks, I've applied it to this, and it works!

Reply:
> If you use ScalingLazyColumn here and adapt the list of Location elements accordingly, it will handle scaling items on and off the screen as you scroll. See https://youtu.be/B7D3G6tC9n0?t=697.

Thanks, it works.